packages/core/stories/checkbox/checkbox.stories.tsx

```diff
@@ -248,17 +248,6 @@ export const LongTextGroup: StoryFn<typeof CheckboxGroup> = (args) => {
         "Checkboxes allow the user to select multiple options from a set. If you have multiple options appearing in a list, you can preserve space by using checkboxes instead of on/off switches. If you have a single option, avoid using a checkbox and use an on/off switch instead.",
       value: "checkboxes",
     },
-    {
-      label:
-        "Radio buttons allow the user to select one option from a set. Use radio buttons for exclusive selection if you think that the user needs to see all available options side-by-side. Radio buttons allow the user to select one option from a set. Use radio buttons for exclusive selection if you think that the user needs to see all available options side-by-side.",
-      value: "radio",
-    },
-    {
-      disabled: true,
-      label:
-        "On/off switches toggle the state of a single settings option. The option that the switch controls, as well as the state it’s in, should be made clear from the corresponding inline label. Switch can also be used with a label description thanks to the FormControlLabel component.",
-      value: "switches",
-    },
```

Comment:
is this supposed to be a group with only one checkbox? was there a reason for removing these?

Reply:
you're right doesn't make sense to have a group with only a single component - have added back